_database-integrations/dynamodb/dynamodb-v1.md

```diff
@@ -173,10 +173,24 @@ replication-sections:
     anchor: "log-based-incremental-replication"
     summary: "Details about Log-based Incremental Replication via DynamoDB streams."
     content: |
-      This integration supports Full Table and Log-based replication. Log-based replication is done through DynamoDB streams, not Amazon Kinesis streams. In order to use streams, they must be enabled on the tables created in DynamoDB. When in your AWS account, go to the `manage streams` section and choose `New Image` or `New and old images`. If one of these aren't selected, the integration will fail during sync. This is handled on a table-by-table basis, so you must enable streams on each.
+      Stitch's DynamoDB integration uses [DynamoDB Streams](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Streams.html) to perform Log-based Incremental Replication. To use Log-based Incremental Replication, streams must be enabled on every table in DynamoDB you want to replicate using this Replication Method. 
```

Comment:
For external links, please add a `:{target="new"}` attribute at the end of the link. Ex:

```
[Copy that displays on the page](https://www.some-url.com):{target="new"}
```

This opens the link in a new page. It's useful so people don't have to click back a bunch of times when looking through our docs.